tests/test_trade.py

```diff
@@ -51,25 +61,47 @@ def run_base_trade_test(self, user_policies, config_file, additional_overrides=N
             override_dict.update(additional_overrides)
         simulator.setup_simulated_entities(override_dict)
         simulator.run_simulation()
+        # comment this to view the generated log files
+        file_loc = logging.getLogger().handlers[0].baseFilename
+        os.remove(file_loc)
```

Comment:
instead of forcing us to comment out a line, shouldn't we move this file wipe step at the beginning of logging, and leave the file on the hard drive in between runs by default?